src/main/java/org/jabref/gui/importer/actions/AppendDatabaseAction.java

```diff
@@ -173,4 +120,78 @@ private static void mergeFromBibtex(JabRefFrame frame, BasePanel panel, ParserRe
         panel.getUndoManager().addEdit(ce);
         panel.markBaseChanged();
     }
+
+    /**
+     * Adds the specified node as a child of the current root. The group contained in <b>newGroups </b> must not be of
+     * type AllEntriesGroup, since every tree has exactly one AllEntriesGroup (its root). The <b>newGroups </b> are
+     * inserted directly, i.e. they are not deepCopy()'d.
+     */
+    private static void addGroups(GroupTreeNode newGroups, CompoundEdit ce) {
```

Comment:
There is commented-out code in this method and if I see it correctly, `ce` is never used. So please delete this variable.